dateparser/custom_language_detection/fasttext_wrapper.py

```diff
@@ -0,0 +1,34 @@
+import fasttext
+import numpy as np
+
+
+class FastTextWrapper:
+    def __init__(self, model_path):
+        self.model = fasttext.load_model(model_path)
+
+    def predict(self, text, k=1, threshold=0.0, on_unicode_error="strict"):
+        def check(entry):
+            if entry.find("\n") != -1:
+                raise ValueError("predict processes one line at a time (remove '\\n')")
+            entry += "\n"
+            return entry
+
+        if isinstance(text, list):
+            text = [check(entry) for entry in text]
+            all_labels, all_probs = self.model.f.multilinePredict(
+                text, k, threshold, on_unicode_error
+            )
+            return all_labels, all_probs
+        else:
+            text = check(text)
+            predictions = self.model.f.predict(text, k, threshold, on_unicode_error)
+            if predictions:
+                probs, labels = zip(*predictions)
+            else:
+                probs, labels = ([], ())
+
+            return labels, np.asarray(probs)
```

Comment:
We got some error in the test, and it is relative to numpy and this line, so I created this part of the code just copy pasting from the library 
https://github.com/facebookresearch/fastText/blob/1142dc4c4ecbc19cc16eee5cdd28472e689267e6/python/fasttext_module/fasttext/FastText.py#L239
but instead 
`return labels, np.array(probs, copy=False)` I written `return labels, np.asarray(probs)`